plugins/eval.py

```diff
@@ -0,0 +1,110 @@
+import time
+
+from util import hook, http
+
+
+def fetch_languages():
+    lang_list = http.get_json("https://api.judge0.com/languages")
+    m = {x["name"].split()[0].lower(): x["id"] for x in lang_list}
+    m.pop("text")
+    m.pop("executable")
+    m["python2"] = [x["id"] for x in lang_list if x["name"].startswith("Python (2")][0]
+    m["py"] = m["py3"] = m["python3"] = m["python"]
+    m["py2"] = m["python2"]
+    m["clj"] = m["clojure"]
+    m["cpp"] = m["c++"]
+    m["csharp"] = m["c#"]
+    m["node"] = m["js"] = m["javascript"]
+    m["rb"] = m["ruby"]
+
+    return m
+
+
+languages = fetch_languages()
```

Comment:
initialize languages and statuses inside of runcode, not at global scope. importing the module shouldn't cause network activity